irohad/model/registration/command_registration.hpp

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright Soramitsu Co., Ltd. 2017 All Rights Reserved.
+ * http://soramitsu.co.jp
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef IROHA_COMMAND_REGISTRATION_HPP
+#define IROHA_COMMAND_REGISTRATION_HPP
+
+#include "common/class_handler.hpp"
+
+// ----------| commands |----------
+#include "model/commands/add_asset_quantity.hpp"
+#include "model/commands/add_peer.hpp"
+#include "model/commands/add_signatory.hpp"
+#include "model/commands/assign_master_key.hpp"
+#include "model/commands/create_account.hpp"
+#include "model/commands/create_asset.hpp"
+#include "model/commands/create_domain.hpp"
+#include "model/commands/remove_signatory.hpp"
+#include "model/commands/set_permissions.hpp"
+#include "model/commands/set_quorum.hpp"
+#include "model/commands/transfer_asset.hpp"
+
+/**
+ * File contains registration for all command subclasses
+ */
+
+namespace iroha {
+  namespace model {
+
+    static ClassHandler command_handler;
```

Comment:
https://stackoverflow.com/a/3837513
Not sure if there is a profit of using static in this case. We can put `command_handler` inside `CommandRegistry`, and then create objects of this class where it is required. Same applies to other registries